frontend/src/components/Accessibility/AccessibilityProvider.tsx

```
@@ -0,0 +1,223 @@
+      }
+
+      switch (event.key) {
+        case 'Tab':
+          // Ensure focus indicators are visible when using keyboard
+          if (!showFocusIndicators) {
+            setShowFocusIndicators(true);
+          }
+          break;
+        case 'Escape':
+          // Close modals, dropdowns, etc.
+          const escapeEvent = new CustomEvent('accessibility-escape');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)